File: java/src/com/v2soft/styxlib/library/messages/StyxRClunkMessage.java
package com.v2soft.styxlib.library.messages;

import java.io.IOException;

import com.v2soft.styxlib.library.io.StyxDataReader;
import com.v2soft.styxlib.library.messages.base.StyxMessage;
import com.v2soft.styxlib.library.messages.base.enums.MessageType;

public class StyxRClunkMessage extends StyxMessage {
	
	public StyxRClunkMessage(int tag)
	{
		super(MessageType.Rclunk, tag);
	}
	
    @Override
    public void load(StyxDataReader input) 
        throws IOException  {
    }	
	@Override
	public int getBinarySize() {
		return super.getBinarySize();
	}

	@Override
	protected String internalToString() {
		return null;
	}
	
}

File: cpp/lib/messages/StyxRClunkMessage.cpp
/*
 * StyxRClunkMessage.cpp
 *
 *  Created on: Jun 3, 2012
 *      Author: mrco
 */

#include "messages/StyxRClunkMessage.h"

StyxRClunkMessage::StyxRClunkMessage(StyxTAG tag) :
	StyxMessage(Rclunk, tag){
}

StyxRClunkMessage::~StyxRClunkMessage() {
}
// =======================================================
// Virtual methods
// =======================================================
void StyxRClunkMessage::load(IStyxDataReader *input) {
	return;
}


